include/zephyr/senss/senss_sensor_virtual.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef SENSS_SENSOR_VIRTUAL_H_
+#define SENSS_SENSOR_VIRTUAL_H_
+
+#include <senss/senss_sensor.h>
+#include <senss/senss_datatypes.h>
+
+/**
+ * @brief Virtual Sensor API
+ * @addtogroup senss_virtual
+ * @{
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @struct senss_sensor_virtual_api
+ * @brief Virtual sensor callback api
+ *
+ * A virtual sensor must register this callback API during sensor register.
+ */
+struct senss_sensor_virtual_api {
```

Comment:
Since `Virtual` sensor focus on fusion and computing,  many `Physical` sensor's properties may not supported,  and also `Virtual` sensor may have its specific APIs,  so,  just defined separated APIs,   to make more clear for different vendors and developers,  let algorithm vendors and developers focus on `Virtual` sensor API, and sensor chip vendors and developers focus on `Physical` sensor API.

If set_range/get_range is useful in some `Virtual` sensors just like your example, will add into `Virtual` sensor API.